CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.5.1)
+project(CMaizeSuperbuild VERSION 1.0.0)
+
+include(ExternalProject)
+
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/src)
+include_directories(${EXTERNAL_INSTALL_LOCATION}/Shared-Library)
```

Comment:
Why are these here?

Reply:
They were from the older push I had, I kept them since I had the two folder in my project, but for the superbuild I will just use an external project add command to add the directories in from the repository.